examples/production/values.yaml

```diff
@@ -41,7 +23,17 @@ spire-server:
       drop: [ALL]
     seccompProfile:
       type: RuntimeDefault
+
+  dataStorage:
+    enabled: true
+    size: 10Gi
```

Comment:
The db only seems to be used for a very small number of entries, even on a big cluster. 10Gi seems overkill.
1Gi? 
I'd recommend even smaller, but I've seen provisioning break on volumes < 1Gi.